Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -452,27 +454,31 @@ def get_endpoints(self,
             endpoints = reply.get('reply').get('endpoints', [])
         return endpoints
 
-    def isolate_endpoint(self, endpoint_id):
+    def isolate_endpoint(self, endpoint_id, incident_id=None):
+        request_data = {
+            'endpoint_id': endpoint_id
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
everywhere else too
```suggestion
            'endpoint_id': endpoint_id,
```

Reply:
Done